### PR TITLE
cleaner living for pal

### PR DESCRIPTION
**Pull request by howard-stearns**
Cleanup some accumulated cruft in pal code.
Addresses some errors in log, but does not attempt to fix other damage that has been done along the way, such as whitespace.
May or may not help with the "blank entry problem"

### CODE REVIEW

**Comment by howard-stearns** [comment]
test plan:
- smoke. 
- check log for errors related to pal.js or Pal.qml.
- confirm that display looks ok wrt entry [display name, username/fingerprint, isAdmin], both when you are admin and when you are not.

File: interface/resources/qml/hifi/Pal.qml [hunk 1]
```diff
@@ -438,6 +437,7 @@ Rectangle {
             right: parent.right
         }
     }
+    }
```
Comment (howard-stearns) [inline interface/resources/qml/hifi/Pal.qml]: The close at 429 was making the keyboard not have the parent that it should.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 6e3ae1fe4fdb788ed51703cf86aaa2ec4abfc035 available at: https://gist.github.com/c3cd433390a7203f1efdb1a6585cfc66<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9689-6e3ae1f.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9689-6e3ae1f.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 6e3ae1fe4fdb788ed51703cf86aaa2ec4abfc035 available at: https://gist.github.com/2e859b2ace2377af31d72f873b8b1b03<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9689-6e3ae1f.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9689-6e3ae1f.dmg

**Comment by Nex-Pro** [comment]
@howard-stearns Does this refer to the blank entries I observed and mentioned when there were more than 100 avatars at the stress test. If so should this be tested during the next one ?

File: interface/resources/qml/hifi/Pal.qml (hunk 1 above)
Comment (davidkelly) [inline interface/resources/qml/hifi/Pal.qml]: oh nice catch!

**Comment by davidkelly** [comment]
I didn't test in 100 avatar domains, but this passed my PAL smoke + a perusal of the logs showed no errors.

**Pull request merged**